src/pynwb/data/nwb.ophys.yaml

```diff
@@ -116,6 +116,7 @@ groups:
     doc: Signals from ROIs
     dtype: float32
     name: data
+    quantity: '?'
```

Comment:
Can you explain what this is for? I'm confused about why data would be optional for a structure that's supposed to hold time series data, or how it fits in with the rest of the changes.

Reply:
Correct me if I'm wrong, but I believe the idea is that data can either be stored in `data` or can be stored in `external_file`. If it's in an external file, then `data` should not be required. The changes to the API ensure that at least one of those is present. If this is correct, maybe I can add this to the documentation on nwb-schema to avoid confusion.

Reply:
That's correct, but it looks like this change is on RoiResponseSeries, not ImageSeries. Did you mean to add this change around here? 
https://github.com/NeurodataWithoutBorders/pynwb/blob/96b7f081534f26ca1a8a3f3080a6faae914fe5f3/src/pynwb/data/nwb.image.yaml#L24

Reply:
Oh yes I must have meant that. I'll correct it. By the way, would there be an efficient way to change the order of these yaml files? Maybe I'm the only one, but I find alphabetical order very confusing.

Reply:
You're not alone... it's objectively very annoying! Way back, I tried to to get ruamel to output things in a different order, and couldn't get it to work, so I punted. I think I remember someone making an issue, and adding the correct ruamel code to get the sensible ordering.... I'll see if I can find it and add it to NWB 2.0 milesone

Reply:
OK, glad we agree on that and it's good to hear that someone has made an attempt. If you end up finding it I can help usher it through.

Reply:
https://github.com/NeurodataWithoutBorders/nwb-schema/issues/139